source/elements/oneTBB/source/task_scheduler/attach_tag_type.rst

```diff
@@ -0,0 +1,24 @@
+.. SPDX-FileCopyrightText: 2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+===============
+attach tag type
+===============
+**[scheduler.attach]**
+
+An ``attach`` tag type is specifically used with ``task_arena`` and 
+``task_scheduler_handle`` interfaces. It is guaranteed to be default constructible.
+
+.. code:: cpp
+
+    namespace oneapi {
+        namespace tbb {
+            using attach = /* unspecified */
```

Comment:
What if we decide to move it again somewhere else? `using` seems easier to provide backward compatibility (currently, we need to deprecate `task_arena::attach` because we cannot reassign it to `tbb::attach`.

Reply:
Agree, the conversation can be marked as resolved.